test/QuestFactory.spec.ts

```diff
@@ -120,8 +120,19 @@ describe('QuestFactory', () => {
     })
 
     it('Should revert if trying to use existing quest id', async () => {
-      expect(
```

Comment:
this test wasn't actually testing anything, so I fixed it here.

Reply:
Ahhhhh thank you 😅🫡